i18n/en.json

```diff
@@ -1815,6 +1815,53 @@
   "admin.reload.reloadDescription.featureName": "Reload Configuration from Disk",
   "admin.reload.reloadDescription.recycleDatabaseConnections": "Environment > Database > Recycle Database Connections",
   "admin.reload.reloadFail": "Reloading unsuccessful: {error}",
+  "admin.reporting.workspace_optimization.access.description": "Web server settings could be affecting access.",
+  "admin.reporting.workspace_optimization.access.site_url.description": "Your webserver settings are not passing a live URL test, this would prevent users from accessing this workspace, we recommend updating your settings.",
+  "admin.reporting.workspace_optimization.access.site_url.title": "Misconfigured Web Server",
+  "admin.reporting.workspace_optimization.access.title": "Workspace Access",
+  "admin.reporting.workspace_optimization.chip_problems": "Problems: {count}",
+  "admin.reporting.workspace_optimization.chip_suggestions": "Suggestions: {count}",
+  "admin.reporting.workspace_optimization.chip_warnings": "Warnings: {count}",
+  "admin.reporting.workspace_optimization.configuration.description": "You have configuration problems to resolve",
+  "admin.reporting.workspace_optimization.configuration.ssl.title": "Configure SSL to make your server more secure",
+  "admin.reporting.workspace_optimization.configuration.title": "Configuration",
+  "admin.reporting.workspace_optimization.configureSessionLength": "Configure Session Length",
+  "admin.reporting.workspace_optimization.configureSSL": "Configure SSL",
+  "admin.reporting.workspace_optimization.configureWebServer": "Configure Web Server",
+  "admin.reporting.workspace_optimization.data_privacy.description": "Get better insight and control over your data.",
+  "admin.reporting.workspace_optimization.data_privacy.retention.description": "A lot of organizations in highly regulated indsutries require more control and insight with thier data. Become more aware and take control of your data by trying out data retention and compliance features.",
+  "admin.reporting.workspace_optimization.data_privacy.retention.title": "Become more data aware",
+  "admin.reporting.workspace_optimization.data_privacy.title": "Data Privacy",
+  "admin.reporting.workspace_optimization.downloadUpdate": "Download Update",
+  "admin.reporting.workspace_optimization.ease_of_management.description": "We have suggestions that could make your managemenet easier.",
+  "admin.reporting.workspace_optimization.ease_of_management.guests_accounts.description": "We noticed several accounts using different domains from your Site URL. Gain more control over what other organizations can access with the guest account feature.",
```

Comment:
answer: user accounts that differ from the site-url

Reply:
@cwarnermm for this one i think the end of the sentence should simply be "We recommend enabling guest accounts" eliminating the words of starting a trial because this message would show after they have already started a trial? wouldnt it?